network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -39,11 +39,11 @@
   </div>
 </div>
 
-{% with section_class="p-md-5 col-12 col-lg-10 offset-lg-1" %}
+{% with section_class_col="col-12 col-lg-10 offset-lg-1" section_class_padding_x="px-md-5" section_class_padding_y="py-md-5" %}
```

Comment:
I had to break this into 3 vars as we can't override Bootstrap classes that are there for the same breakpoints

e.g., `p-md-5 p-md-0` --> `p-md-5` overrides `p-md-0`